### ISSUE TITLE
intel-hybrid-driver does not work with libva 2.0

### ISSUE THREAD

**Issue by eclipseo** [01org/intel-hybrid-driver#25]
When using the new libva, the hybrid driver fails to start:

```
$  vainfo
libva info: VA-API version 1.1.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib64/dri/i965_drv_video.so
libva info: Found init function __vaDriverInit_1_1
/usr/lib64/dri/hybrid_drv_video.so init failed
Failed to wrapper hybrid_drv_video.so
libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.1 (libva 2.1.0)
vainfo: Driver version: Intel i965 driver for Intel(R) Skylake - 2.1.0
```

**Comment by YYchenli** [comment]
maybe your driver have some question

**Comment by montvid** [comment]
The 1.0.2-11.fc28 driver does not work with fedora 28 and libva 2.1.0. Anyway to fix this?
`$ vainfo
libva info: VA-API version 1.1.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib64/dri/i965_drv_video.so
libva info: va_openDriver() returns -1
vaInitialize failed with error code -1 (unknown libva error),exit
`

**Comment by montvid** [comment]
Does the driver need intel-vaapi-driver i.e. i965_drv_video.so? I would like to use hybrid_drv_video.so instead of i965_drv_video.so...

**Comment by chivakker** [comment]
You can use hybrid_drv_video.so only with i965_drv_video.so. Check intel-vaapi-driver for configuration necessary.

Depending on how you compile libva-2.0, you might need to modify for libva-2.0 to load.  I'll submit a PR for it.

**Comment by montvid** [comment]
Ok, I compiled intel-vaapi-driver with meson 'enable_hybrid_codec=true' but I don't see in vainfo if hybrid_drv_video.so is loaded but there are no errors in vainfo so maybe everything is fine and this bug report can be closed. I did not compile libva, I installed libva 2.1 from fedora 28.

So far I can remember for decoding hw video i965_drv_video.so is enough. For what is hybrid_drv_video.so used?

**Comment by chivakker** [comment]
Yes decoding/encoding/post-processing works with i965_drv_video.so.  I'd need to ask you what you're trying to do and why you think you need the hybrid_drv_video.so.

**Comment by montvid** [comment]
Well I was thinking as I can't install i965_drv_video.so because it is not in fedora repo (I know I can install it via rpmfusion) because of license issues maybe hybrid_drv_video.so can do hardware decode of vp8/vp9 instead of i965_drv_video.so? Am I wrong?

**Comment by chivakker** [comment]
No, that's not the scope of hybrid driver.  Readme can provide details.

**Comment by montvid** [comment]
@chivakker from intel readme "Hybrid VP9 Decoder" from fedora repo "It allows to accelerate VP9 videos on Skylake and Kabylake architectures." Sorry I am not a native speaker so maybe there is something between the lines I don't understand. Thanks for the info you provided.

**Comment by eclipseo** [comment]
The PR #26 fixes the issue for me.

@montvid As the Fedora maintainer I will push an update with the patch, you will be able to install it with the updates-testing repository tomorrow / in two days. Let me know if it's okay for you.
